ballerina/receiver.bal

```diff
@@ -117,9 +117,9 @@ public isolated client class MessageReceiver {
     isolated remote function complete(@display {label: "Message"} Message message)
                                         returns Error? {
         if message?.lockToken.toString() != DEFAULT_MESSAGE_LOCK_TOKEN {
-            return complete(self, message?.lockToken.toString());
+            return complete(self, message);
         }
-        return createError("Failed to complete message with ID " + message?.messageId.toString());
+        return createError(string `Failed to complete message with ID ${message?.messageId.toString()}`);
```

Comment:
```suggestion
        return createError(string `Failed to complete message with ID: ${message?.messageId.toString()}`);
```

Reply:
@RDPerera Lets update all the similar places

Reply:
Adressed in [99aa177](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/196/commits/99aa1775c6f483964ef25fc150705d42f6ee0bdb)